widgetssdk/src/main/java/com/glia/widgets/UiTheme.kt

```diff
@@ -30,10 +30,6 @@ import kotlinx.parcelize.Parcelize
         "The remote configurations approach is more versatile and better suited for future development."
 )
 data class UiTheme(
-    /**
-     * Text to be shown on the top of the app bar in the chat
-     */
-    val appBarTitle: String? = null,
```

Comment:
This should also be a separate PR. Also, the release notes should be mentioned in the PR section as breaking changes. Same for the removed deprecated properties so that we could communicate this in the 3.0.0 release and migration guides